noxfile.py

```diff
@@ -117,14 +117,18 @@ def lint(session):
 @nox.session
 def format(session):
     install_groups(session, include=["format"], include_self=False)
-    session.run("codespell", "--enable-colors")
+    session.run(
+        "codespell", "--enable-colors", "--skip=./PyPI_analysis/results/*.ipynb"
```

Comment:
Because in our jupyter notebook, there are many plots displaying, and in the file, it shows in hash, and `codespell` will complain.
For example,
```
./PyPI_analysis/results/results.ipynb:270: BA ==> BY, BE
```
and so on.